dianna/dashboard/pages/Time_series.py

```diff
@@ -51,25 +60,75 @@
         contributing positively towards the classification decision are
         indicated in red and those who contribute negatively in blue.
         """)
+    elif load_example == "Scientific case: FRB":
+        ts_file = (data_directory / 'FRB211024.npy')
+        ts_model_file = (model_directory /
+                        'apertif_frb_dynamic_spectrum_model.onnx')
```

Comment:
The frb model `apertif_frb_dynamic_spectrum_model.onnx` does not exist in the [`model_directory`](https://github.com/dianna-ai/dianna/tree/790-add-scientific-use-case-frb-to-dashboard/dianna/models). Can you please add the model? Later, when the downloader functionality is implemented, this part can be refactored.